tests/snmp/test_snmp_interfaces.py

```diff
@@ -6,26 +6,39 @@
 ]
 
 @pytest.mark.bsl
-def test_snmp_interfaces(duthosts, rand_one_dut_hostname, localhost, creds):
```

Comment:
>test_snmp_interfaces [](start = 4, length = 20)

You changed function name, but it is used in several places already.

Reply:
Changed function name to as it was before.